src/types/redis_bloom_chain.cc

```diff
@@ -214,8 +214,8 @@ rocksdb::Status BloomChain::InsertCommon(engine::Context &ctx, const Slice &user
       }
       std::string data = bf_data_list.back().ToString();
       bloomAdd(item_hash_list[i], data);
-      *bf_data_list.back().GetSelf() = std::move(data);
```

Comment:
I reallized why this is wrong, since it's a bit hack, if `rocksdb::PinnableSlice` is a pinned slice ( `pinned_ == true` ), it's `GetSelf` returns nullptr.

However `PinSlice(data, nullptr)` might not a good idea for this, since it would leak `data` after it dtor.